component/class/defaults.yml

```diff
@@ -52,7 +52,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.86.0
+        tag: support_keycloak_25
```

Comment:
Don't forget to adjust once it's released

Reply:
Done